commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaService.java

```diff
@@ -56,4 +65,29 @@ public boolean currentUserIsInProtectedGroup() {
   public String getMfaSystem() {
     return mfaSystem;
   }
+
+  public void switchMfaSystem(String mfaSystem) {
+    switch (mfaSystem) {
+      case "OTP" :
+        this.mfaSystem = DEFAULT_MFA_SYSTEM;
+        break;
+      case "Fido 2" :
+        this.mfaSystem = MFA_FIDO;
+        break;
+      case "SuperGluu" :
+        this.mfaSystem = MFA_OIDC;
+        break;
+      default:
+        this.mfaSystem =DEFAULT_MFA_SYSTEM;
+        break;
+    }
+  }
+
+  public boolean isMfaFeatureActivated() {
+    Boolean isMfaFeatureAcivated = featureService.isActiveFeature(MFA_FEATURE);
+    if(isMfaFeatureAcivated == null) {
+      return true;
+    }
+    return isMfaFeatureAcivated;
```

Comment:
FeatureService does not return null
```suggestion
    return featureService.isActiveFeature(MFA_FEATURE);
```